templates/python-tests/.travis.yml

```diff
@@ -4,6 +4,7 @@ services:
   - docker
 
 before_script:
+  - export APP_IMAGE=keboola-component
   - docker -v
   - docker build -t my-component .
```

Comment:
a tady se ta proměnná nevyužívá :/

Reply:
potřebuje ji ten deploy script který se volá dále.

Reply:
omg, sorry jsem slepej.

Reply:
fixnu to použití

Reply:
Jo, to jo. Ale měla by být i na tom řádku 9 :)